### DISCUSSION TITLE
unlicensed

### DISCUSSION THREAD

**Discussion by Chacalbill** [aclap-dev/video-downloadhelper#1529] [Help and Q&A]
### Your question

I purchased a license on March 29, 2021, but the extension appears as unlicensed.

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 4,
    "successfull_dl": 1
}

**Comment by jcfrog** [comment]
hi,
you probably just need to revlidate your license key: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
